.github/workflows/testbuild.yml

```diff
@@ -1,6 +1,12 @@
 ---
 name: Test Build
-on: [pull_request]  # yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
I think this should not run when performing a pull request, only with a push to master.

Reply:
The testbuild is supposed to run on pull requests. This is the behavior from before. I added if conditions starting from [Line 47](https://github.com/freifunk-berlin/bbb-configs/pull/684/files#diff-fb5c483c388bb8411369b8ae18c01c14b2389999d00b8f6cdd898a27d8282cd1R47) that differentiate between PR and push

Reply:
sorry, missed that one...